### DISCUSSION TITLE
20230828 Spotify returning Invalid Grant

### DISCUSSION THREAD

**Discussion by uipe** [rern/rAudio#1405] [Bug Reports]
Hi fresh user to your application, I am trying to enable spotify but I am getting Invalid Grant error. Any way to debug / get more information to report back ?

**Comment by SpaceCudet** [comment]
I was getting the same with the latest version (20230828). Rolled back to 20230620 and it now works.

**Reply by Roland-B** [reply]
Same here. Latest builds have broken adding key to Spotify, also if restoring from previous backups the key file has been renamed in newer build from "spotify" to "spotifykey" so would no longer work.

**Reply by rern** [reply]
The bug was found. A fix will be in the next update.

**Reply by filipe-be1s** [reply]
Thanks for the feedback.
